src/redprl/signature_mini.sml

```diff
@@ -96,7 +96,8 @@ struct
           case psort of 
             RedPrlSortData.HYP tau =>
               let
-                val RedPrlParameterTerm.VAR v = ptm
+                val v = case ptm of RedPrlParameterTerm.VAR v => v
+                           | _ => raise Fail "Hypothesis renaming failed: not a variable"
```

Comment:
Oops, I mean "let's keep the code the way *you* wrote it". So, I mean that I like your adding the case statement.